andela/trime
# Organization page projects

#### What does this PR do?
Shows all the projects in the organization page instead of just showing three.

#### Description of Task to be completed?
Stop rendering only the first three projects in the project array and render everything in the array.

#### How should this be manually tested?
Create more than three projects in an organization. When the visit the organization's page, you'll see all the projects, even those beyond the third project.

#### What are the relevant pivotal tracker stories?
[User cannot view more than 3 projects  in an organization.](https://www.pivotaltracker.com/story/show/120181463)
